src/PostgREST/Functions.hs

```diff
@@ -0,0 +1,183 @@
+{-# LANGUAGE OverloadedStrings #-}
+module PostgREST.Functions
```

Comment:
Functions is a very generic name. Maybe something like DatabaseHelpers or QueryBuilder would be more appropriate